node_exporter/node_exporter.spec

```diff
@@ -41,17 +49,27 @@ getent passwd prometheus >/dev/null || \
 exit 0
 
 %post
+%if 0%{?el7}
```

Comment:
You still need to perform similar actions on EL6, like registering or unregistering the service. On uninstall it should also be stopped if it's running.

Reply:
Updated

Reply:
Updated